packages/loot-core/src/server/api.ts

```diff
@@ -600,6 +600,32 @@ handlers['api/payee-delete'] = withMutation(async function ({ id }) {
   return handlers['payees-batch-change']({ deleted: [{ id }] });
 });
 
+handlers['api/rules-get'] = async function () {
+  checkFileOpen();
+  return handlers['rules-get']();
+};
+
+handlers['api/payee-rules-get'] = async function ({ id }) {
+  checkFileOpen();
+  return handlers['payee-rules-get'](id);
+};
+
+handlers['api/rule-create'] = withMutation(async function ({ rule }) {
+  checkFileOpen();
+  const addedRule = await handlers['rule-add'](rule);
+  return (addedRule as { id }).id;
```

Comment:
Since the other create functions seem to just return the string ID, I tried to mimic that here.  But that means if there was an `error` set, not only is that error message not propagated to the client but now this type cast should throw a different error.  This probably is not what we want, but I was not sure what to do here.

Reply:
Anything we could do here to remove the manual type coercion?